frontend/simple/sass/theme/default.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../node_modules/bulma/sass/utilities/initial-variables";
+@import "../../node_modules/bulma/sass/utilities/functions";
+
+$primary: #5dc8f0;
+$secondary: #a0d10e;
+$tertiary: #f89201;
+
+$success: saturate($secondary, 10%);
+$warning: lighten(adjust-hue($tertiary, 15%), 0%);
+$danger: lighten(adjust-hue($tertiary, -30%), 20%);
+$info: $primary;
+
+$grey: lighten(#000, 60%);
+
+// $purple: #847bb9;
+// $red: #ed6c63;
+// $turquoise: $blue-dark;
+// $yellow: #fce473;
+
+@import "../../node_modules/bulma/sass/utilities/derived-variables";
+
+$addColors: (
+  "secondary":($secondary, #fff),
+  "tertiary": ($tertiary, #fff),
+);
+
+$colors: map-merge($colors, $addColors);
+
+// Typography
+
+$link: $primary;
+$text: $grey-darker;
+
+$family-sans-serif: "Helvetica Neue", "Helvetica", "Arial", sans-serif;
+$family-monospace: #{monospace};
+
+// Miscellaneous
+
+$body-background-color: #fff;
+$radius: 3px;
+$gap: 16px;
```

Comment:
I would expect files within the `theme` folder to just be theme files.

And the above does not look like *structure* css, but as we discussed on the call "colors and typography", so therefore I'm assuming this is the "kindergarten professional theme". If so, it needs to be named as such, not `default.scss`. The "default" theme must be set by a variable/setting in JavaScript somewhere. See also recently created issue #388.

Reply:
Your are right, I've moved `$radius` and `$gap` to `bulma_overrides/utilities`, so this file, no named `theme/kindergarten-professional.scss` keeps only the variables related to the theme.

Reply:
> BTW, did you see the notice at the top of that post?
>
>> UPDATE April 2016: I wrote up a new approach for this with fallbacks [CSS4 Variables with Fallbacks Using Sass](https://codepen.io/jakealbaugh/post/css4-variables-with-fallbacks-using-sass).
>
> Thoughts on that?

Yes I did, and it's similar to those changes that we need to integrate on Bulma. Meanwhile, I've created [an issue on Bulma](https://github.com/jgthms/bulma/issues/1837) asking if they have a date in mind to add  CSS Variables support, since it's on their roadmap. Maybe we can wait for it before forking it. What do you think?